pkg/cmd/step/scheduler/test_data/step_scheduler_config_migrate/nongitops_advanced/config.yaml

```diff
@@ -1027,9 +1027,62 @@ push_gateway: {}
 sinker: {}
 tide:
   merge_method:
-    jenkins-x-apps/jx-app-test-lifecycle: rebase
+    cloudbees/cb-app-slack: merge
```

Comment:
Just a thought, is it expected that there's a lot of new repos being added to the test?  I just wondered if there's an issue with the integration test getting real prow config from a cluster rather than test data?

Reply:
They were already there - they just didn't have a merge method specified.

Reply:
yay!